targets/lenovo-x1-carbon.nix

```diff
@@ -120,6 +120,7 @@
                     macAddress = "02:00:00:03:03:05";
                     ramMb = 3072;
                     cores = 4;
+                    cid = 4;
```

Comment:
is there anyway to script this as e.g. ++cid so as to avoid hard coding for each VM?

Reply:
Sure, I'll check how to do it.

Reply:
Added auto-assigning but at the same time kept the ability to override with a static value.